tests/test_band_pass_filter.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+import numpy as np
+
+import matplotlib.pyplot as plt
+
+from audiomentations.augmentations.transforms import BandPassFilter
+
+
+class TestBandPassFilter(unittest.TestCase):
+    def test_band_pass_filter(self):
+        sample_rate = 16000
+        t = .25                # signal duration in sec
+        f = 500                # signal frequency in Hz
+        samples = np.arange(t * f, dtype=np.float32) / sample_rate
+        samples = np.sin(2 * np.pi * f * samples)
+
+        augment = BandPassFilter(min_center_freq=100, 
+                                 max_center_freq=5000,
+                                 min_q=1.0,
+                                 max_q=2.0,
+                                 p=1.0)
+        processed_samples = augment(
+            samples=samples, sample_rate=sample_rate
+        )
+
+        self.assertEqual(processed_samples.shape, samples.shape)
+        self.assertEqual(processed_samples.dtype, np.float32)
+        
+        plt.plot(samples)
+        plt.plot(processed_samples, '-.')
+        plt.show()
```

Comment:
Please wrap plotting code in tests inside something like an `if DEBUG:` block. You can put the DEBUG constant (default to False) below the imports of the file. When needed, one can turn this on manually and look at the plots.

Motivation: The test runner (e.g. the one on Github) may get stuck while trying to run the test suite if it should show plots while running the tests

Reply:
Done